src/components/UserMenu/__tests__/UserMenu.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { act } from 'react-test-renderer';
+import UserMenu from 'components/UserMenu';
+import { MemoryRouter, Route } from 'react-router';
+import { buddyMenuDetails, mockSchemaTypes, newbieMenuDetails } from '__mocks__';
+import AuthContext, { AuthContextData } from 'contexts/AuthContext';
+import waitForExpect from 'wait-for-expect';
+import { render, fireEvent } from '@testing-library/react';
+import { MockedProvider } from '@apollo/react-testing';
+jest.mock('@material-ui/core/IconButton', () => 'IconButton');
+jest.mock('@material-ui/icons/Close', () => 'CloseIcon');
+jest.mock('@material-ui/core/Box', () => 'Box');
+jest.mock('@material-ui/core/Drawer', () => 'Drawer');
+jest.mock('@material-ui/core/Divider', () => 'Divider');
+jest.mock('@material-ui/core/CircularProgress', () => 'CircularProgress');
+jest.mock('components/UserMenuDetails/UserMenuDetails', () => 'UserMenuDetails');
+jest.mock('components/UserMenuBuddy/UserMenuBuddy', () => 'UserMenuBuddy');
+jest.mock('components/UserMenuNewbies/UserMenuNewbies', () => 'UserMenuNewbies');
+jest.mock('components/UserMenuSettings/UserMenuSettings', () => 'UserMenuSettings');
+jest.mock('components/UserMenuSettings/UserMenuSettings', () => 'UserMenuSettings');
+jest.mock('buddy-app-schema', () => ({
+  UserRole: {
+    Newbie: 'NEWBIE',
+    Buddy: 'BUDDY',
+  },
+}));
```

Comment:
Since [PR-39](https://github.com/BuildIt-Poland/buddy-app/pull/39) is merged this mock is unnecessary